src/appellate/appellate.js

```diff
@@ -36,6 +41,13 @@ AppellateDelegate.prototype.handleCaseSelectionPage = async function () {
 // check every link in the document to see if RECAP has it
 AppellateDelegate.prototype.attachRecapLinksToEligibleDocs = async function () {
   // filter the links for the documents available on the page
+  APPELLATE.createDummyIframe('dummyframe');
+
+  let form = document.getElementsByName('doDocPostURLForm');
+  if (form.length) {
+    form[0].setAttribute('target', 'dummyframe');
+  }
```

Comment:
Per discussion today on Slack, it seems like this and related code can be simplified by just removing the `target` attribute of the `doDocPostURLForm` form.